src/Workspaces/VisualBasic/Portable/Classification/SyntaxClassification/EmbeddedLanguageTokenClassifier.vb

```diff
@@ -0,0 +1,25 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.Classification
+Imports Microsoft.CodeAnalysis.PooledObjects
+Imports Microsoft.CodeAnalysis.VisualBasic.EmbeddedLanguages.LanguageServices
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Classification.Classifiers
+    Friend Class EmbeddedLanguageTokenClassifier
+        Inherits AbstractSyntaxClassifier
+
+        Public Overrides ReadOnly Property SyntaxTokenKinds As ImmutableArray(Of Integer) = ImmutableArray.Create(Of Integer)(SyntaxKind.StringLiteralToken)
+
+        Public Overrides Sub AddClassifications(workspace As Workspace, token As SyntaxToken, semanticModel As SemanticModel, result As ArrayBuilder(Of ClassifiedSpan), cancellationToken As CancellationToken)
```

Comment:
Consider breaking this extra long line

Reply:
Maybe this method could be pulled into the parent type (`AbstractSyntaxClassifier`) and only having a language-specific implementation for `VisualBasicEmbeddedLanguageProvider.Instance`?

Reply:
> Maybe this method could be pulled into the parent type (AbstractSyntaxClassifier) 

So, unfortunately "AbstractSyntaxClassifier" is not what is sounds like.  it's not an abstract impl shared across VB and C#.  Instead, there are two AbstractSyntaxClassifiers.  Once defined in VB and one defined in C#.  They're more appropriately thought of as AbstractCSharpSyntaxClassifier and AbstractVisualBasicSyntaxClassifier.  They don't have anything in common to share.

Reply:
Ok.  i my understanding was correct, but the design here was due to the classifier stuff being written long before we had any shared compiler code between C#/VB.  Turns out there was no need for separate langauge versions, so i merged them and just took your good suggestion.  Thanks!